static/app/views/explore/savedQueries/savedQueriesTable.tsx

```diff
@@ -49,17 +49,24 @@ export function SavedQueriesTable({mode, perPage}: Props) {
     exclude: mode === 'owned' ? 'shared' : 'owned', // Inverse because this is an exclusion
     perPage,
   });
+  const filteredData = data?.filter(row => row.query?.length > 0) ?? [];
   const {deleteQuery} = useDeleteQuery();
   const renderBodyCell = (col: Column, row: SavedQuery) => {
+    const query = row.query[0];
+    // This shouldn't happen, but adding this to enforce type safety
+    if (query === undefined) {
+      return null;
+    }
     if (col.key === 'name') {
       return (
         <NoOverflow>
           <Link
             to={getExploreUrl({
               organization,
               ...row,
+              ...query,
```

Comment:
Your UI doesn't currently work for multi-query mode right? That's why we're just expanding the contents of a single query?

Reply:
Yup this is just for a single query. I have a follow up PR for multi queries